cdap-ui/app/services/alert-on-valium/alert-on-valium.js

```diff
@@ -18,34 +18,37 @@ angular.module(PKG.name + '.services')
   .factory('myAlertOnValium', function($alert, $window) {
     var isAnAlertOpened = false,
         alertObj;
-     function show(obj) {
-       if (!isAnAlertOpened) {
-         isAnAlertOpened = true;
-         alertObj = $alert(obj);
-         alertObj.$scope
-          .$on('alert.hide', function() {
-            isAnAlertOpened = false;
-          });
-       } else {
+    function show(obj) {
+      if (!isAnAlertOpened) {
+        isAnAlertOpened = true;
+
+        obj.duration = obj.type === 'success' ? 3 : false;
```

Comment:
can you put a comment for what this 3 signifies? It'll help if we ever want to change the timeout in the future.